lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -0,0 +1,247 @@
+            editButtons = viewerControls.find('.annotator-edit');
+            delButtons = viewerControls.find('.annotator-delete');
+            closeButtons = viewerControls.find('.annotator-close');
+
+            // Note and ddit, delete and close buttons always come in quadruplets
+            for (i = 0; i < editButtons.length; i++) {
+                tabControls.push($(notes.get(i)));
+                tabControls.push($(editButtons.get(i)));
+                tabControls.push($(delButtons.get(i)));
+                tabControls.push($(closeButtons.get(i)));
+            }
+
+            return tabControls;
+        },
+
+        getEditorTabControls: function () {
+            var editor, editorControls, textArea, save, cancel, tabControls = [];
+
+            // Editor elements
+            editor = this.annotator.element.find('.annotator-editor');
+            editorControls = editor.find('.annotator-controls');
+            textArea = editor.find('.annotator-listing')
+                             .find('.annotator-item')
+                             .first()
+                             .children('textarea');
+            save  = editorControls.find('.annotator-save');
+            cancel = editorControls.find('.annotator-cancel');
+
+            tabControls.push($(textArea.get(0)), $(save.get(0)), $(cancel.get(0)));
+
+            return tabControls;
+        },
+
+        focusOnNextTabControl: function (tabControls, tabControl) {
+            var nextIndex;
+
+            _.each(tabControls, function (element, index) {
+                if (element.is(tabControl)) {
+                    nextIndex = index === tabControls.length - 1 ? 0 : index + 1;
+                    tabControls[nextIndex].focus();
+                }
+
+            });
+        },
+
+        focusOnPreviousTabControl: function (tabControls, tabControl) {
+            var previousIndex;
+            _.each(tabControls, function (element, index) {
+                if (element.is(tabControl)) {
+                    previousIndex = index === 0  ? tabControls.length - 1 : index - 1;
+                    tabControls[previousIndex].focus();
+                }
+            });
+        },
+
+        onHighlightKeyDown: function (event) {
+            var KEY = $.ui.keyCode,
+                keyCode = event.keyCode,
+                target = $(event.currentTarget),
+                annotations, position;
+
+            switch (keyCode) {
+                case KEY.ENTER:
+                case KEY.SPACE:
+                    if (!this.annotator.viewer.isShown()) {
+                        position = target.position();
+                        annotations = target.parents('.annotator-hl').addBack().map(function() {
+                            return $(this).data('annotation');
+                        });
+                        this.annotator.showViewer($.makeArray(annotations), {top: position.top, left: position.left});
```

Comment:
Why do we need to show all notes when we we're using a keyboard?
When we use the mouse, the program does not know what note to show as they are located one above the other. But here we know what the note we want to open. What do you think?